tools/swift-api-digester/swift-api-digester.cpp

```diff
@@ -748,10 +767,20 @@ class PrunePass : public MatchedNodeListener, public SDKTreeDiffPass {
 
   void foundMatch(NodePtr Left, NodePtr Right, NodeMatchReason Reason) override {
     switch (Reason) {
-    case NodeMatchReason::Added:
+    case NodeMatchReason::Added: {
       assert(!Left);
       Right->annotate(NodeAnnotation::Added);
+      if (Ctx.checkingABI()) {
+        // Adding members to a fixed layout type is ABI-breaking.
```

Comment:
Removing members too (even if they're not public).

Reply:
I'll have a separate PR for adding/removing nodes.